packages/core/recommended-gitignore.txt

```diff
@@ -0,0 +1,8 @@
+node_modules
+.env
```

Comment:
`.env.example` should be excluded for demonstration, or submit the `.env` and ignore `.env.local`, `.env.production` etc.

Reply:
`.gitignore` generated by [gitignore](https://www.toptal.com/developers/gitignore/) would help, e.g. https://www.toptal.com/developers/gitignore/api/dotenv,node,windows,macos,linux,git